public/pages/Overview/containers/Overview/Overview.tsx

```diff
@@ -104,11 +107,30 @@ export default class Overview extends Component<OverviewProps, OverviewState> {
     ];
   }
 
+  onTimeChange = ({ start, end }: { start: string; end: string }) => {};
+
+  onRefresh = async () => {};
+
   render() {
     return (
       <ContentPanel title={'Overview'}>
         <EuiFlexGroup direction="column">
-          <EuiFlexItem className="grid-item">
+          <EuiFlexItem>
+            <EuiFlexGroup gutterSize={'s'}>
+              <EuiFlexItem grow={9}>
+                <EuiFieldSearch
+                  fullWidth={true}
+                  // onChange={this.onSearchChange}
+                  placeholder={'Search findings'}
```

Comment:
still needed?

Reply:
Yes, I commented in another file, forgot to add // TODO here